src/com/esotericsoftware/kryo/Kryo.java

```diff
@@ -429,9 +430,9 @@ public Registration register (Class type, Serializer serializer, int id) {
 		return register(new Registration(type, serializer, id));
 	}
 
-	/** Stores the specified registration. If the ID is already in use by the same type, the old entry is overwritten. If the ID is
-	 * already in use by a different type, a {@link KryoException} is thrown. Registering a primitive also affects the
-	 * corresponding primitive wrapper.
+	/** Stores the specified registration. If the ID is already in use by the same type, the old entry is overwritten.
+	 * If the ID is already in use by a different type, undefined behavior may result. Registering a primitive also affects

```

Comment:
Again, I'd say "If the ID is already in use by a different type, undefined behavior may result." can be removed.